lucene/benchmark/src/java/org/apache/lucene/benchmark/byTask/tasks/NewAnalyzerTask.java

```diff
@@ -42,14 +43,13 @@ public NewAnalyzerTask(PerfRunData runData) {
   
   public static final Analyzer createAnalyzer(String className) throws Exception{
     final Class<? extends Analyzer> clazz = Class.forName(className).asSubclass(Analyzer.class);
-    try {
-      // first try to use a ctor with version parameter (needed for many new Analyzers that have no default one anymore
-      Constructor<? extends Analyzer> cnstr = clazz.getConstructor(Version.class);
-      return cnstr.newInstance(Version.LATEST);
```

Comment:
And this removal is because we stopped using `Version` to emulate backwards compatibility for analyzers, long ago also?

Reply:
Yes, I think so too.